### PR TITLE
packer init doc clarification

### PR DESCRIPTION
**Pull request by azr**
Be a little bit clearer about what is going to be installed where, and that init doesn't have a state "like TF"

### CODE REVIEW

**Review by sylviamoss — APPROVED** [review]
Nice!

For v1.7.0 should we also add on this same page that `packer init` will only be able to download multi-plugins?

**Comment by azr** [comment]
@sylviamoss , yup you're right ! I just added these, I hope they're good ! So many notes at the top of that page !

Screenshot 2021-02-03 at 11 49 13

File: website/content/docs/commands/init.mdx [hunk 1]
```diff
@@ -7,13 +7,14 @@ sidebar_title: <tt>init</tt>
 
 # `init` Command
 
--> **Note:** Packer does not currently have the notion of a state.
+-> **Note:** Packer does not currently have the notion of a state like Terraform
+has. In other word, currently `packer init` is only in charge of installing
```
Comment (sylviamoss) [inline website/content/docs/commands/init.mdx:11]:
```suggestion
has. In other words, currently `packer init` is only in charge of installing
```
My Grammarly tool corrects this one to `words`

File: website/content/docs/commands/init.mdx [hunk 2]
```diff
@@ -23,6 +24,13 @@ usage this should not be an issue. Otherwise you can set the
 your personal [access token page](https://github.com/settings/tokens) to
 generate a new token.
 
+-> **Note:** Packer init does not work with legacy JSON templates. You can
+upgrade your JSON configfiles to HCL using the hcl2ugprade command.
```
Comment (sylviamoss) [inline website/content/docs/commands/init.mdx:28]:
```suggestion
upgrade your JSON config files to HCL using the hcl2ugprade command.
```

**Comment by sylviamoss** [comment]
Oh yeah! Lots of notes but necessary ones. This is good!

**Comment by SwampDragons** [comment]
Looking at this, I'm reminded of the adage "if you highlight everything, you've highlighted nothing" I think we need to figure out what one or two messages actually belong in notes. Otherwise people will not read them.

**Comment by SwampDragons** [comment]
My suggestion is we leave "only works in hcl" and "only works with multiplugins" in notes, and leave the rest as normal text.

**Comment by azr** [comment]
Hey @SwampDragons Gotcha, opening a PR

**Comment by sylviamoss** [comment]
Nice feedback!

**Pull request merged**